src/Microsoft.IdentityModel.Tokens/BaseConfiguration.cs

```diff
@@ -10,7 +11,7 @@ namespace Microsoft.IdentityModel.Tokens
     /// <summary>
     ///  Represents a generic metadata configuration which is applicable for both XML and JSON based configurations.
     /// </summary>
-    public abstract class BaseConfiguration
+    public abstract class BaseConfiguration /*: IConfigurationRetrievalTime*/ // L2 TODO: internal until L2 cache is implemented S2S.
```

Comment:
Have you considered just adding a new property to BaseConfiguration in 8.x?

Reply:
It just needs to be commented out right now because the interface is internal right now

Reply:
THis is commented out since we're keeping the interface internal right now.